SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -932,7 +957,7 @@ USpatialNetConnection* USpatialNetDriver::AcceptNewPlayer(const FURL& InUrl, boo
 
 	if (!ErrorMsg.IsEmpty())
 	{
-		UE_LOG(LogNet, Log, TEXT("PreLogin failure: %s"), *ErrorMsg);
+		UE_LOG(LogNet, Error, TEXT("PreLogin failure: %s"), *ErrorMsg);
```

Comment:
Change to `LogSpatialNetDriver` category.

Reply:
Done.